client/src/login/LoginForm.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+export const LoginForm = reduxForm({
```

Comment:
Using exported name 'LoginContainer' as identifier for default export  import/no-named-as-default